mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -960,6 +962,21 @@ func TestAccComposerEnvironment_fixPyPiPackages(t *testing.T) {
 	})
 }
 
+// This bootstraps the IAM roles needed for the service agents when using encryption.
+func grantEncrypterDecrypterRoleToServiceAgents(t *testing.T) {
+	serviceAgents := []string{
+		"cloudcomposer-accounts",
```

Comment:
Currently I'm in the process of figuring out which of these accounts needs the encrypter/decrypter role. For now we can just grant it to all of them since that's what the hardcoded samples originally did.